metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -416,8 +416,13 @@ def process_topic_config_result(
         topic_configurations: dict,
     ) -> None:
         try:
-            assert config_result_future.done()
-            assert config_result_future.exception() is None
+            if not config_result_future.done():
```

Comment:
config_result_future.result() will automatically throw a TimeoutErrror and will also reraise any exception that the underlying method raised - should simplify this code a bunch

Reply:
Thanks, I simplified the code.